lib/ret_web/controllers/api/v1/hub_controller.ex

```diff
@@ -9,6 +11,24 @@ defmodule RetWeb.Api.V1.HubController do
   # Only allow access with secret header
   plug(RetWeb.Plugs.HeaderAuthorization when action in [:delete])
 
+  plug(:authorize_resource, model: Hub, id_field: "hub_sid", only: [:update])
+
+  def update(conn, %{"id" => hub_sid, "hub" => params}) do
```

Comment:
Hadn't realized that you'd want to do name changes as phoenix messages in #107. I can remove this or leave it in if we want the REST API anyway.

Reply:
Let's remove for now, since I can't think of a reason right now to have it